src/Topics/StandardModernPrecision/Bids1C.hs

```diff
@@ -190,8 +190,7 @@ _makeJumpBid level suit = do
     minSuitLength suit 5
     -- This should be your longest suit
     -- TODO: if you're 5-5, which suit do you bid first?
-    mapM_ (forbid . compareSuitLength suit Shorter) . filter (/= suit) $
-        T.allSuits
+    mapM_ (suit `longerThan`) . filter (/= suit) $ T.allSuits
```

Comment:
This one really did change: to avoid the ambiguity of what to do with 5-5 hands, we now require that the intended suit be the _longest_ in the hand. The TODO about supporting 5-5 hands is still there for later cleanup.